project/src/components/app/app.tsx

```diff
@@ -38,23 +41,24 @@ function App(props: ConnectedComponentProps): JSX.Element {
   if (!isDataLoaded) {
     return <Spinner />;
   }
-
+  // Если пользователь авторизован, то при переходе на страницу Sign In выполняется перенаправление на главную страницу. пока непонятно, потом сделать
```

Comment:
лучше делать приставку TODO, тогда легче искать по коду
```suggestion
  //TODO: Если пользователь авторизован, то при переходе на страницу Sign In выполняется перенаправление на главную страницу. пока непонятно, потом сделать
```